pkgs/development/python-modules/appnope/default.nix

```diff
@@ -1,21 +1,27 @@
 { lib
 , buildPythonPackage
-, fetchPypi
+, fetchFromGitHub
+, pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "appnope";
   version = "0.1.3";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "sha256-Ar2RxN6Gn7seHFCq/ECYgnp6VKsvOdncumyVR+2SDiQ=";
+  src = fetchFromGitHub {
+    owner = "minrk";
+    repo = "appnope";
+    rev = version;
+    sha256 = "sha256-JYzNOPD1ofOrtZK5TTKxbF1ausmczsltR7F1Vwss8Sw=";
   };
 
-  meta = {
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  meta = with lib; {
     description = "Disable App Nap on macOS";
-    homepage    = "https://pypi.python.org/pypi/appnope";
-    platforms   = lib.platforms.darwin;
-    license     = lib.licenses.bsd3;
+    homepage = "https://pypi.python.org/pypi/appnope";
```

Comment:
```suggestion
    homepage = "https://pypi.org/project/appnope/";
```

Reply:
Maybe https://github.com/minrk/appnope while we're at it? pypi is just a package distributor, the homepage button on their page also points to that repo.

Reply:
fine for me